packages/twenty-front/src/modules/object-record/record-show/components/RecordShowContainer.tsx

```diff
@@ -183,7 +184,7 @@ export const RecordShowContainer = ({
             {isPrefetchLoading ? (
               <PropertyBoxSkeletonLoader />
             ) : (
-              inlineFieldMetadataItems.map((fieldMetadataItem, index) => (
+              inlineFieldMetadataItems?.map((fieldMetadataItem, index) => (
```

Comment:
why is it optional now?

Reply:
This currently assumes a record has other fields than its label identifier. If you have a joint table for example, it might not be the case. This is debatable from a product standpoint whether this kind of very slim object deserves a show page or now but I don't feel relaxing the requirement here causes any issue